### ISSUE TITLE
Sproutform Migration deleting random columns (m210328_000000_clean_up_default_fields_in_content_table)

### ISSUE THREAD

**Issue by Jones-S** [barrelstrength/craft-sprout-forms#596]
### Description (screenshots, logs, ...)
We have recently worked on a multisite craft installation. We have not touched an existing site using sprout forms but worked on another. Now when deploying we run the craft command `migrate/all`.
This looks like this:

![sprout](https://user-images.githubusercontent.com/7059580/154128802-50eda3ac-a1b9-442f-b4c8-09c194db9950.jpg)

Now the weird thing is, that after running migrate/all our page breaks, respectively some content is missing.
It even seems that the `m210328_000000_clean_up_default_fields_in_content_table` class is doing some weird deletions.
https://github.com/barrelstrength/craft-sprout-forms/blob/593df6c8f19813d41c6a022a5d3038b40246876a/src/migrations/m210328_000000_clean_up_default_fields_in_content_table.php

I don't really understand what it is supposed to to, but we know that it is responsible for the content loss.

1. We deleted the corrupt database
2. imported a backup with content
3. ran `php craft migrate/all`
4. Applied the Sprout Forms migration
5. Only then the content went missing.

Could you somehow find some sense in this behaviour?
Thanks a lot for your feedback.

### Steps to reproduce
This will be difficult. We just have a specific project where this happens...

### Additional info

- Craft version: 3.7.30.1
- PHP version: 7.4.27
- Database driver & version:  
- Plugins & versions: 
```
     "barrelstrength/sprout-fields": "^3.8",
    "barrelstrength/sprout-forms": "3.13.8",
    "craftcms/cms": "3.7.30.1",
    "craftcms/mailgun": "2.0.0",
    "craftcms/redactor": "2.9.1",
    "mmikkel/cp-field-inspect": "1.2.5",
    "nystudio107/craft-seomatic": "3.4.23",
    "ostark/craft-async-queue": "2.3.0",
    "percipioglobal/craft-colour-swatches": "1.5.0.4",
    "percipioglobal/craft-password-policy": "1.1.0",
    "sebastianlenz/linkfield": "2.0.0-rc.1",
    "spicyweb/craft-neo": "2.12.4",
    "verbb/field-manager": "2.2.4",
    "verbb/formie": "1.5.8",
    "verbb/navigation": "1.4.24",
    "vlucas/phpdotenv": "^2.4.0",
    "wrav/oembed": "1.3.13"
```

Edit:

Also when doing the same locally:
1. use DB with content
2. run `craft migrate/all`
3. apply the sproutform migration
4. reload the cp

I get this error:

```PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'content.field_altText_jaarsubs' in 'field list' in /app/vendor/yiisoft/yii2/db/Command.php:1302
Stack trace:
#0 /app/vendor/yiisoft/yii2/db/Command.php(1302): PDOStatement->execute()
#1 /app/vendor/yiisoft/yii2/db/Command.php(1168): yii\db\Command->internalExecute('SELECT `users`....')
#2 /app/vendor/yiisoft/yii2/db/Command.php(424): yii\db\Command->queryInternal('fetch', NULL)
#3 /app/vendor/yiisoft/yii2/db/Query.php(287): yii\db\Command->queryOne()
#4 /app/vendor/craftcms/cms/src/db/Query.php(167): yii\db\Query->one(NULL)
#5 /app/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1635): craft\db\Query->one(NULL)
#6 /app/vendor/craftcms/cms/src/elements/User.php(461): craft\elements\db\ElementQuery->one()
#7 /app/vendor/yiisoft/yii2/web/User.php(697): craft\elements\User::findIdentity(1)
#8 /app/vendor/craftcms/cms/src/web/User.php(522): yii\web\User->renewAuthStatus()
#9 /app/vendor/yiisoft/yii2/web/User.php(198): craft\web\User->renewAuthStatus()
#10 /app/vendor/craftcms/cms/src/web/User.php(259): yii\web\User->getIdentity()
#11 /app/vendor/mmikkel/cp-field-inspect/src/CpFieldInspect.php(159): craft\web\User->getIsAdmin()
#12 /app/vendor/mmikkel/cp-field-inspect/src/CpFieldInspect.php(96): mmikkel\cpfieldinspect\CpFieldInspect->doIt()
#13 [internal function]: mmikkel\cpfieldinspect\CpFieldInspect->mmikkel\cpfieldinspect\{closure}(Object(yii\base\Event))
#14 /app/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Object(Closure), Object(yii\base\Event))
#15 /app/vendor/yiisoft/yii2/base/Component.php(642): yii\base\Event::trigger('craft\\services\\...', 'afterLoadPlugin...', Object(yii\base\Event))
#16 /app/vendor/craftcms/cms/src/services/Plugins.php(277): yii\base\Component->trigger('afterLoadPlugin...')
#17 /app/vendor/craftcms/cms/src/base/ApplicationTrait.php(1487): craft\services\Plugins->loadPlugins()
#18 /app/vendor/craftcms/cms/src/web/Application.php(93): craft\web\Application->_postInit()
#19 /app/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init()
#20 /app/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#21 [internal function]: yii\base\Application->__construct(Array)
#22 /app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#23 /app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\web\\Appli...', Array, Array)
#24 /app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\web\\Appli...', Array, Array)
#25 /app/vendor/craftcms/cms/src/Craft.php(61): yii\BaseYii::createObject(Array, Array)
#26 /app/vendor/craftcms/cms/bootstrap/bootstrap.php(246): Craft::createObject(Array)
#27 /app/vendor/craftcms/cms/bootstrap/web.php(51): require('/app/vendor/cra...')
#28 /app/web/index.php(20): require('/app/vendor/cra...')
#29 {main}

Next yii\db\Exception: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'content.field_altText_jaarsubs' in 'field list'
The SQL being executed was: SELECT `users`.`password`, `elements`.`id`, `elements`.`fieldLayoutId`, `elements`.`uid`, `elements`.`enabled`, `elements`.`archived`, `elements`.`dateCreated`, `elements`.`dateUpdated`, `elements_sites`.`id` AS `siteSettingsId`, `elements_sites`.`slug`, `elements_sites`.`siteId`, `elements_sites`.`uri`, `elements_sites`.`enabled` AS `enabledForSite`, `elements`.`canonicalId`, `elements`.`dateLastMerged`, `users`.`username`, `users`.`firstName`, `users`.`lastName`, `users`.`email`, `users`.`unverifiedEmail`, `users`.`admin`, `users`.`locked`, `users`.`pending`, `users`.`suspended`, `users`.`lastLoginDate`, `users`.`lockoutDate`, `users`.`photoId`, `users`.`hasDashboard`, `content`.`id` AS `contentId`, `content`.`field_addressLink`, `content`.`field_address`, `content`.`field_enjAddress`, `content`.`field_active`, `content`.`field_alternativePageTitle`, `content`.`field_altText_jaarsubs`, `content`.`field_badgeStatus`, `content`.`field_badgeText`, `content`.`field_enjAltText_xaafgynq`, `content`.`field_enjCaption_kwikipqq`, `content`.`field_heroText_euyoqdmz`, `content`.`field_cookieText`, `content`.`field_anchor`, `content`.`field_darkBackground`, `content`.`field_highlightDarkBackground`, `content`.`field_facebookLink`, `content`.`field_colorPicker_bywzzakz`, `content`.`field_gdprText`, `content`.`field_googleLink`, `content`.`field_enjHeroToggle_iqhbahsb`, `content`.`field_enjHeroText_pgiuxeok`, `content`.`field_linkCollectionTitleIcon_bidfmwlu`, `content`.`field_enjQuicklinkToggle_qwkpifxo`, `content`.`field_instagramLink`, `content`.`field_enjRichTextSmallToggle_leitlggl`, `content`.`field_contactRichtext_rmovsded`, `content`.`field_formContact`, `content`.`field_contactInfo`, `content`.`field_linkStyle_iausiilb`, `content`.`field_enjLinkType_lqdccptx`, `content`.`field_formLobTadel`, `content`.`field_logoSize_rxfdlcpc`, `content`.`field_lunchgateMenu`, `content`.`field_calendar_vffmuoky`, `content`.`field_newsTextUnibe_ltttdsek`, `content`.`field_newsletterLink_mqomfxws`, `content`.`field_formNewsletter`, `content`.`field_enjOpeningHours_msflmrxr`, `content`.`field_openingHours`, `content`.`field_openingHoursContent`, `content`.`field_openingHoursRichtext_qnqxxkhv`, `content`.`field_primaryColor_atyxjcnv`, `content`.`field_review`, `content`.`field_richtext_bcybzrog`, `content`.`field_pageType`, `content`.`field_secondaryColor_mluuljyj`, `content`.`field_selectIconUnibe_jvhvttcr`, `content`.`field_seo`, `content`.`field_enjTeaserImageSize_lrrvllff`, `content`.`field_enjTeaserLayout_ksddgorg`, `content`.`field_enjTeaserText_seqdfxgh`, `content`.`field_teaserTextUnibe_dyyzjynm`, `content`.`field_enjTeaserTitle_xdpaqycr`, `content`.`field_baseText`, `content`.`field_enjRichText_nwjyjbbr`, `content`.`field_imageHeroText_rfjmawub`, `content`.`field_sectionText`, `content`.`field_simpleRedactor_rfhcwdmw`, `content`.`field_simpleText_jpygzerx`, `content`.`field_imageHeroTextColor_pxbnglqp`, `content`.`field_textColor_zjybbsga`, `content`.`field_sectionTitle`, `content`.`field_baseTitle`, `content`.`field_baseToggle_iqpacdjg`, `content`.`field_tripadvisorLink`, `content`.`field_teaserVariantToggleUnibe_gkxzgiha`, `content`.`field_video_sbnaqzib`
FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId`
FROM `vc_elements` `elements`
INNER JOIN `vc_users` `users` ON `users`.`id` = `elements`.`id`
INNER JOIN `vc_elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id`
INNER JOIN `vc_content` `content` ON (`content`.`elementId` = `elements`.`id`) AND (`content`.`siteId` = `elements_sites`.`siteId`)
WHERE (`elements_sites`.`siteId`=1) AND (`elements`.`id`=1) AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`elements`.`draftId` IS NULL) AND (`elements`.`revisionId` IS NULL)
ORDER BY `users`.`username`
LIMIT 1) `subquery`
INNER JOIN `vc_users` `users` ON `users`.`id` = `subquery`.`elementsId`
INNER JOIN `vc_elements` `elements` ON `elements`.`id` = `subquery`.`elementsId`
INNER JOIN `vc_elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId`
INNER JOIN `vc_content` `content` ON `content`.`id` = `subquery`.`contentId`
ORDER BY `users`.`username` in /app/vendor/yiisoft/yii2/db/Schema.php:676
Stack trace:
#0 /app/vendor/yiisoft/yii2/db/Command.php(1307): yii\db\Schema->convertException(Object(PDOException), 'SELECT `users`....')
#1 /app/vendor/yiisoft/yii2/db/Command.php(1168): yii\db\Command->internalExecute('SELECT `users`....')
#2 /app/vendor/yiisoft/yii2/db/Command.php(424): yii\db\Command->queryInternal('fetch', NULL)
#3 /app/vendor/yiisoft/yii2/db/Query.php(287): yii\db\Command->queryOne()
#4 /app/vendor/craftcms/cms/src/db/Query.php(167): yii\db\Query->one(NULL)
#5 /app/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1635): craft\db\Query->one(NULL)
#6 /app/vendor/craftcms/cms/src/elements/User.php(461): craft\elements\db\ElementQuery->one()
#7 /app/vendor/yiisoft/yii2/web/User.php(697): craft\elements\User::findIdentity(1)
#8 /app/vendor/craftcms/cms/src/web/User.php(522): yii\web\User->renewAuthStatus()
#9 /app/vendor/yiisoft/yii2/web/User.php(198): craft\web\User->renewAuthStatus()
#10 /app/vendor/craftcms/cms/src/web/User.php(259): yii\web\User->getIdentity()
#11 /app/vendor/mmikkel/cp-field-inspect/src/CpFieldInspect.php(159): craft\web\User->getIsAdmin()
#12 /app/vendor/mmikkel/cp-field-inspect/src/CpFieldInspect.php(96): mmikkel\cpfieldinspect\CpFieldInspect->doIt()
#13 [internal function]: mmikkel\cpfieldinspect\CpFieldInspect->mmikkel\cpfieldinspect\{closure}(Object(yii\base\Event))
#14 /app/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Object(Closure), Object(yii\base\Event))
#15 /app/vendor/yiisoft/yii2/base/Component.php(642): yii\base\Event::trigger('craft\\services\\...', 'afterLoadPlugin...', Object(yii\base\Event))
#16 /app/vendor/craftcms/cms/src/services/Plugins.php(277): yii\base\Component->trigger('afterLoadPlugin...')
#17 /app/vendor/craftcms/cms/src/base/ApplicationTrait.php(1487): craft\services\Plugins->loadPlugins()
#18 /app/vendor/craftcms/cms/src/web/Application.php(93): craft\web\Application->_postInit()
#19 /app/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\web\Application->init()
#20 /app/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#21 [internal function]: yii\base\Application->__construct(Array)
#22 /app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#23 /app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\web\\Appli...', Array, Array)
#24 /app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\web\\Appli...', Array, Array)
#25 /app/vendor/craftcms/cms/src/Craft.php(61): yii\BaseYii::createObject(Array, Array)
#26 /app/vendor/craftcms/cms/bootstrap/bootstrap.php(246): Craft::createObject(Array)
#27 /app/vendor/craftcms/cms/bootstrap/web.php(51): require('/app/vendor/cra...')
#28 /app/web/index.php(20): require('/app/vendor/cra...')
#29 {main}
Additional Information:
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'content.field_altText_jaarsubs' in 'field list'
)
```

And you can see that the sproutforms migration dropped that column before:
>     > drop column field_altText_jaarsubs from table {{%content}} ... done (time: 0.092s)

**Comment by BenParizek** [comment]
@Jones-S The reason this migration exists is due to a bug where we were accidentally creating columns in the content table when we were initializing new fields in the Sprout Forms form builder. The intention of the migration is to try to clean those columns up so while we definitely tried to be defensive in our code here it seems, if this is a bug in our code, you have a scenario where something about your column name trips up our logic.

It may be easier to talk through this and troubleshoot on a screenshare so feel free to reach out and [book a time to chat](https://barrelstrengthdesign.com/schedule/ben-parizek/) if that would be helpful.

Currently, I don't see how we'd be dropping that field. Our loop that drops the errant fields looks for fields with the following pattern: `field_defaultField[XYZ]`

https://github.com/barrelstrength/craft-sprout-forms/blob/593df6c8f19813d41c6a022a5d3038b40246876a/src/migrations/m210328_000000_clean_up_default_fields_in_content_table.php#L57-L63

And should only be dropping columns that match that pattern (and your field `field_altText_jaarsubs` doesn't include `defaultField` in its name). When we do drop a column we log it, so you should be able to search 'Dropped column' and see a line like the following in the logs if it was being dropped by Sprout Forms: `Craft::info("Dropped column $customColumnName from " . Table::CONTENT);`

**Comment by Jones-S** [comment]
Hey @BenParizek 
Thanks for getting back to me. Thanks for the offer for a chat, I will get back to that, when I find more time, but that will probably only be after March... 
I don't understand why our fields match the pattern...
Where would I find the logs though? I could quickly check those. Maybe this will bring us one step further...

Thank you.

**Comment by BenParizek** [comment]
Craft will log this to /storage/logs/web.log

I typically delete the contents of web.log before the action I'm troubleshooting. Take the action to generate the log. Then you can search the log for "Dropped column" for that error message or "[error" for any errors or "sprout" for more general stuff that our plugins may have done and see what you find.

**Comment by mijewe** [comment]
I am also having this issue.

The migration is deleting a bunch of fields that don't start with `field_defaultField`

Looks to me like the `if (!strpos($customColumnName, $orphanFieldPrefix) === 0) { ` line is the problem.

Doing a bit of sanity debugging, I think this is a brackets issue.
It's currently evaluating "(NOT strpos) = 0"
But it wants to be evaluatng "NOT (strpos=0)"

because
```
// this will === 0 if the column name starts with field_defaultField
// this will === false if the column name doesn't start with field_defaultField
$strpos = strpos($customColumnName, $orphanFieldPrefix);

// this will === true either way, since !false is true, and != 0 is true
$notstrpos = !$strpos;
```

On my fields that don't start with field_defaultField:
```
$strposEqualsZero = strpos($customColumnName, $orphanFieldPrefix) === 0; // false
$not_strposEqualsZero = !$strposEqualsZero; // true

$actualCheck = !strpos($customColumnName, $orphanFieldPrefix) === 0; // false
```

**Comment by anthonyjharrison** [comment]
Adding another voice to this issue running craft `3.7.47.1` and sprout forms `3.13.8`

I have a field `field_defaultField1` which the migration deletes but then proceeds to delete all the other fields after that. The fields preceding that field seem unaffected. (that's just my take).

terminal output - as you can see first item is correct, the next 6 are incorrect.

```
*** applying m210328_000000_clean_up_default_fields_in_content_table
    > drop column field_defaultField1 from table {{%content}} ... done (time: 0.876s)
    > drop column field_confirmationParagraph_shtalley from table {{%content}} ... done (time: 0.812s)
    > drop column field_confirmationHeading_ugcmzpmb from table {{%content}} ... done (time: 0.892s)
    > drop column field_creditCardFormText_xyyknzyj from table {{%content}} ... done (time: 0.991s)
    > drop column field_packageSegmentTitle_fxdcyjax from table {{%content}} ... done (time: 0.888s)
    > drop column field_displayCustomField_rfgbiaeu from table {{%content}} ... done (time: 0.891s)
    > drop column field_customFieldText_ldurtvrp from table {{%content}} ... done (time: 0.872s)
*** applied m210328_000000_clean_up_default_fields_in_content_table (time: 6.305s)

```

entry in `console.log` for correct column removal

```
2022-07-08 14:24:10 [-][-][-][info][yii\db\Command::execute] ALTER TABLE `content` DROP COLUMN `field_defaultField1`
2022-07-08 14:24:10 [-][-][-][profile begin][yii\db\Command::execute] ALTER TABLE `content` DROP COLUMN `field_defaultField1`
2022-07-08 14:24:11 [-][-][-][profile end][yii\db\Command::execute] ALTER TABLE `content` DROP COLUMN `field_defaultField1`
2022-07-08 14:24:11 [-][-][-][info][application] Dropped column field_defaultField1 from {{%content}}
2022-07-08 14:24:11 [-][-][-][info][yii\db\Command::query] SHOW FULL COLUMNS FROM `content`
2022-07-08 14:24:11 [-][-][-][profile begin][yii\db\Command::query] SHOW FULL COLUMNS FROM `content`
2022-07-08 14:24:11 [-][-][-][profile end][yii\db\Command::query] SHOW FULL COLUMNS FROM `content`
```

entry in log for one of the incorrect removals

```

2022-07-08 14:24:11 [-][-][-][info][yii\db\Command::execute] ALTER TABLE `content` DROP COLUMN `field_confirmationParagraph_shtalley`
2022-07-08 14:24:11 [-][-][-][profile begin][yii\db\Command::execute] ALTER TABLE `content` DROP COLUMN `field_confirmationParagraph_shtalley`
2022-07-08 14:24:12 [-][-][-][profile end][yii\db\Command::execute] ALTER TABLE `content` DROP COLUMN `field_confirmationParagraph_shtalley`
2022-07-08 14:24:12 [-][-][-][info][application] Dropped column field_confirmationParagraph_shtalley from {{%content}}
2022-07-08 14:24:12 [-][-][-][info][yii\db\Command::query] SHOW FULL COLUMNS FROM `content`
2022-07-08 14:24:12 [-][-][-][profile begin][yii\db\Command::query] SHOW FULL COLUMNS FROM `content`
2022-07-08 14:24:12 [-][-][-][profile end][yii\db\Command::query] SHOW FULL COLUMNS FROM `content`

```

**Comment by benjamin-smith** [comment]
Happening on an install of ours as well, the below columns are being dropped.

"craftcms/cms": "3.7.44"
"barrelstrength/sprout-forms": "3.13.8",

```
field_linkedinUrl_wrksqwbs
field_firstName_jelxflrk
field_lastName_thqbcszg
field_gradientWrapperEnd_yddpybgd
field_newsEventEndDate_xminwgvl
field_newsEventLocation_ovnhqqno
field_newsEventStartDate_bvodtxld
```

**Comment by darwinsoftware** [comment]
> I am also having this issue.
> 
> The migration is deleting a bunch of fields that don't start with `field_defaultField`
> 
> Looks to me like the `if (!strpos($customColumnName, $orphanFieldPrefix) === 0) { ` line is the problem.
> 
> Doing a bit of sanity debugging, I think this is a brackets issue. It's currently evaluating "(NOT strpos) = 0" But it wants to be evaluatng "NOT (strpos=0)"
> 
> because
> 
> ```
> // this will === 0 if the column name starts with field_defaultField
> // this will === false if the column name doesn't start with field_defaultField
> $strpos = strpos($customColumnName, $orphanFieldPrefix);
> 
> // this will === true either way, since !false is true, and !0 is true
> $notstrpos = !$strpos;
> ```
> 
> On my fields that don't start with field_defaultField:
> 
> ```
> $strposEqualsZero = strpos($customColumnName, $orphanFieldPrefix) === 0; // false
> $not_strposEqualsZero = !$strposEqualsZero; // true
> 
> $actualCheck = !strpos($customColumnName, $orphanFieldPrefix) === 0; // false
> ```

This is the correct analysis and fix. Quite an annoying problem if you don't have the ability to edit the migration file before migrate/all is executed (e.g. if migrate/all is executed immediately as part of an automated deployment workflow). 

In our case, we pulled the database to development during a short content freeze, fixed the migration as per the above, ran the migration on dev, and then pushed the database back up. This prevents the broken migration from being applied again the next time migrate/all is executed.

**Comment by michaelfromtheoutfit** [comment]
Hey all!

I'm having the exact same problem.

I'm planning to do exactly what @darwinsoftware mentioned above - but I'm curious if @BenParizek released a new version with that particular migration corrected and we first upgraded to that version of the plugin, would that solve the problem also?

I feel like this issue is going to keep cropping up for people.

**Comment by mijewe** [comment]
I get the feeling this plugin is somewhat abandoned. There is a PR for the fix but it doesn't seem like it'll get merged in.

https://github.com/barrelstrength/craft-sprout-forms/pull/614

**Comment by anthonyjharrison** [comment]
Yeah I get the same feeling although their roadmap states they're readying for Craft 4...  https://sprout.barrelstrengthdesign.com/docs/support/road-map.html#commercial-plugins Maybe there's hope... @BenParizek any thoughts on progressing a fix for this?

**Comment by BenParizek** [comment]
I'm adding this bug to the top of my list to review and will try to have a fix out soon. 

The [Road Map](https://sprout.barrelstrengthdesign.com/docs/support/road-map.html) (recently updated with a few more comments) continues to be the best summary of where things stand as I continue on the long road to catching up and getting back to a more responsive level of support.

**Comment by BenParizek** [comment]
I've just released Sprout Forms v3.13.9 which should resolve this migration. Let me know if you continue to run into issues.

This issue seems to have been introduced when Craft started appending random strings to the content table columns in 3.7.0.
